app/components/Forms/InscriptionForm.tsx

```diff
@@ -89,15 +89,15 @@ export default function InscriptionForm() {
           id="password"
           label="Entrez votre mot de passe"
           placeholder="Mot de passe..."
-          pattern="^(?=.*[A-Za-z])(?=.*\d)[!#$&%*+=?|\-A-Za-z\d]{8,}$"
+          pattern={process.env.NEXT_PUBLIC_PASS_REGEXP}
           errorMessage="La saisie ne réponds pas aux exigences de sécurité."
           ref={PWRef}
         />
         <TextInputRef
           type={!isPWVisible ? 'password' : undefined}
           id="password"
           placeholder="Confirmer votre mot de passe..."
-          pattern="^(?=.*[A-Za-z])(?=.*\d)[!#$&%*+=?|\-A-Za-z\d]{8,}$"
+          pattern={process.env.NEXT_PUBLIC_PASS_REGEXP}
```

Comment:
`pattern` does not share the same syntax as JS string regexp for some reason. You can either convert the regex to the correct format or create a function that check the field onChange. That's why we don't have the same behavior on APP/API.